pkg/kube/inject/webhook.go

```diff
@@ -852,7 +857,7 @@ func parseInjectEnvs(path string) map[string]string {
 			env = strings.ToUpper(k) // if not found, use the custom env directly
 		}
 		if env != "" {
-			newEnvs[env] = res[i+1]
+			newEnvs[env] = strings.Replace(res[i+1], "%2F", "/", -1)
```

Comment:
I think this might not work because the go uri.URI class already urldecodes the path before this function is called. From the golang doc:
```
Note that the Path field is stored in decoded form: /%47%6f%2f becomes /Go/. A consequence is that it is
impossible to tell which slashes in the Path were slashes in the raw URL and which were %2f. This distinction
is rarely important, but when it is a client must use other routines to parse the raw URL or construct the parsed
URL. For example, an HTTP server can consult req.RequestURI, and an HTTP client can use URL{Host:
"example.com", Opaque: "//example.com/Go%2f"} instead of URL{Host: "example.com", Path: "/Go/"}.
```

Reply:
Use :ENV: as separator as we have discussed via slack.